README.rst

```diff
@@ -73,10 +73,9 @@ Windows
 
 .. code-block:: console
 
-    pip install virtualenv
-    virtualenv <your-env>
+    py -3.9 -m venv <your-env>
```

Comment:
If `py` is the script from [this project](https://pypi.org/project/py/), please note that its page says:

> NOTE: this library is in maintenance mode and should not be used in new code.